API_Quick_Start.md

```diff
@@ -0,0 +1,258 @@
+
+One simple way to have the access token without copy-pasting is to keep it in a variable:
+
+```
+export TOKEN=$(http --form POST https://data-hub.eglobalmark.com/auth/realms/datahub/protocol/openid-connect/token client_id=<client_id> client_secret=<client_secret> grant_type=client_credentials | jq -r .access_token)
+```
+
+Then to simply use it the HTTP requests:
+
+```
+http https://data-hub.eglobalmark.com/... Authorization:"Bearer $TOKEN" ...
+```
+
+For brevity and clarity, the `Authorization` header is not displayed in the sample HTTP requests described below.
+
+## Case study
+
+This case study is written for anyone who wants to get familiar with the API, we use a real example to make it more concrete.
+
+We will create the following entities:
+- A beekeeper
+- An apiary
+- A beehive
+- Sensors linked to the beehive which measures two metrics (temperature and humidity)
+
+* We start by creating the beekeeper, the apiary, the beehive and sensors 
+```
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities < samples/beekeeper.jsonld
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities < samples/apiary.jsonld
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities < samples/sensor_temperature.jsonld
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities < samples/sensor_humidity.jsonld
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities < samples/beehive.jsonld
+```
+
+* (optional) We can delete the created entities and recreate them in batch:
+   
+```
+http DELETE https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:Beekeeper:01
+http DELETE https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:Apiary:01
+http DELETE https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:Sensor:01
+http DELETE https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:Sensor:02
+http DELETE https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01
+
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entityOperations/create < samples/apiculture_entities.jsonld
+```
+
+* The created beehive can be retrieved by id:
+
+```
+http https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01
+```
+
+* Or by querying all entities of type BeeHive:
+
+```
+http https://data-hub.eglobalmark.com/ngsi-ld/v1/entities type==BeeHive Link:"<https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld>; rel=http://www.w3.org/ns/json-ld#context; type=application/ld+json" Content-Type:application/json

```

Comment:
I have a suggestion : splitting too long commands in multi line.
For example :
```
http https://data-hub.eglobalmark.com/ngsi-ld/v1/entities \
type==BeeHive \
Link:"<https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld>; rel=http://www.w3.org/ns/json-ld#context; type=application/ld+json" \
Content-Type:application/json
```
This can be split differently, I just think the actual format doesn't help to clearly identify what's done in the request.